modules/Kconfig

```diff
@@ -50,6 +49,9 @@ comment "Unavailable modules, please install those via the project manifest."
 # config ZEPHYR_<MODULE_NAME_UPPER>_MODULE
 # 	bool
 
+comment "nordic module not available."
```

Comment:
Maybe:
```suggestion 
comment "Hal nordic module not available."
```

Reply:
Oh, I thought I had already corrected this. Thanks for catching it.
I'll use `hal_nordic`, to stay consistent with was is displayed when the module is available.

Reply:
Fixed.